### ISSUE TITLE
Issue when importing data

### ISSUE THREAD

**Issue by jmsluna** [camflint/JourneyToDayOneImporter#3]
Hi @camflint,

First place I wanted to thank you for the effort put in this tool, since Journey and Day One don't put it very easy to export files to other apps.

I have followed the steps, if I recall correctly, but the result gives me consistently these errors:

''WARNING: coordinates are invalid: 1.7976931348623157e+308 1.7976931348623157e+308''

''Use help command for more information on how use use the dayone CLI.
ERROR: 
Error: Invalid value(s) for option --coordinate: 40.925469299501174''.

The result: 37 succeeded, 379 failed, 1 skipped.

The zip is has been extracted from Mac Journey 4.1.21, last version.
¿Do you know what may have gone wrong?

Thanks in advance!

**Comment by connerxyz** [comment]
**Seeing this warning means that the code processed an entry that didn't have location information.**

Turns out that Journey exports lat, lon coordinates with this value whenever your entries don't have location information (it's essentially `null` or `None`). 

_An example from my entries:_

<img width="381" alt="Screen Shot 2022-08-11 at 11 41 44 PM" src="https://user-images.githubusercontent.com/3486466/184286388-33bbabed-51be-4527-8780-a1ca095345e0.png">

Further, this value is is the max float value for some programming languages, like Python.

```
>>> import sys
>>> sys.float_info
sys.float_info(max=1.7976931348623157e+308, ...)
```

**Comment by jmsluna** [comment]
Thank you @connerxyz. What I don't fully understand is why these entries aren't processed and added into my diary. Can't they be added as entries without location?

**Comment by connerxyz** [comment]
I’m fairly certain entries missing location info ARE added to Day One (i. e, not skipped). There is just an print message set to warn you that it doesn’t seem the location information is valid (i.e., is missing). I’ll double check this.
